packages/base/source/server/events.js

```diff
@@ -5,7 +5,7 @@ Space.messaging.define(Space.messaging.Event, 'Todos', {
   },
 
   TodoCreated: {
-    id: Guid,
+    todoId: Guid,
```

Comment:
maybe, but remember the aggregate is never referred to with a simple `id` prop, it's always either the target of a command `targetId` or the source of an event `sourceId`. In the context of the message it should be obvious what `id` refers to with events like `TodoCreated`. The only way I would _not_ go with plain `id` is if there is a benefit with automated mapping. Let's review this again @DominikGuzei